react-stopwatch/my-app/src/Stopwatch.tsx

```diff
@@ -0,0 +1,40 @@
+import { useState } from 'react';
+import './App.css';
+import './Stopwatch.css';
+import { FaPlay, FaPause } from 'react-icons/fa';
+
+export function Stopwatch() {
+  const [time, setTime] = useState(0);
+  const [intervalFunction, setIntervalFunction] = useState<NodeJS.Timeout>();
+  const running = intervalFunction !== undefined;
```

Comment:
In the Q&A I forgot to show you this trick! A fun way to write this is:
`const running = !!intervalFunction;`

It looks magic, but it's not. The first `!` returns `true` if `intervalFunction` is falsey. The second one toggles it. So `!!intervalFunction` is `true` if `intervalFunction` is truthy (defined) and `false` otherwise.